web-app/src/components/common/Errorbox/index.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { PropTypes as PT } from 'prop-types';
+import { ErrorContainer } from './styled';
+
+const Errorbox = props => {
+  return props.error ? (
```

Comment:
Nicer to do an implicit return, and also drop the ternary for `props.error && ...`